frontend/js/src/user/playlists/Playlists.tsx

```diff
@@ -175,8 +178,26 @@ export default class UserPlaylists extends React.Component<
                 <FontAwesomeIcon icon={faPlusCircle as IconProp} size="2x" />
                 <span>Create new playlist</span>
               </div>
-            </Card>
-          )}
+            </Card>,
+            <Card
+              key="import-playlist"
+              className="import-playlist"
+              data-toggle="modal"
+              data-target="#ImportPlaylistModal"
+              onClick={() => {
+                NiceModal.show(ImportPlaylistModal)
+                  // @ts-ignore
```

Comment:
Making a note here, which can definitely be pushed back for a future refactor if you don't want to touch it:
We can improve the types here and avoid forcefully ignoring the typescript error like so:

```tsx
NiceModal.show<
  JSPFPlaylist, // <- this is the expected return type
  NiceModalHocProps
>(ImportPlaylistModal)
.then((playlist) => { // note the removed type here, no longer needed
  this.onPlaylistCreated(playlist); // <- playlist here has the expected type of JSPFPlaylist that we set above
});
```